tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core

```

Comment:
(I've not had time follow this thread properly.) Regarding the namespaces,
is it the case that `FakeItEasy.Core` is a namespace that test authors
never to have to use?
On 2 Mar 2016 12:17 pm, "Blair Conrad" <notifications@github.com> wrote:

> In tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs
> <https://github.com/FakeItEasy/FakeItEasy/pull/605#discussion_r54710317>:
>
> > @@ -0,0 +1,146 @@
> > +﻿namespace FakeItEasy.Tests.Core
>
> And ICallOrderingContext (I'm embarrassed to ask.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FakeItEasy/FakeItEasy/pull/605/files#r54710317>.
>